### PR TITLE
Add bindings for workspaces and layouts

### PR DESCRIPTION
**Pull request by CestDiego**
C-tab C-S-tab for workspaces (like tabs [vim tabs?])
C-s-tab C-S-s-tab for layouts (using super key like a WM)

Also ( M-s- | C-s ) + [0-9]  to change to number [0-9] workspace/layout

### CODE REVIEW

File: layers/+window-management/eyebrowse/packages.el [hunk 1]
```diff
@@ -59,6 +59,24 @@
...
                     "[r] rename"))))))
 
+      (global-set-key (kbd "<C-tab>")
+                      'spacemacs//workspaces-eyebrowse-next-window-config-l)
+      (global-set-key (kbd "<C-S-tab>")
+                      'spacemacs//workspaces-eyebrowse-prev-window-config-h)
+      (global-set-key (kbd "<C-S-iso-lefttab>")
+                      'spacemacs//workspaces-eyebrowse-prev-window-config-h)
+
+      (global-set-key (kbd "M-s-1") 'spacemacs//workspaces-eyebrowse-switch-to-window-config-0-0)
```
Comment (StreakyCobra) [inline layers/+window-management/eyebrowse/packages.el]: I think your numbering is wrong somewhere here ;-)

**Comment by StreakyCobra** [comment]
Also, should it be documented somewhere?

**Comment by robbyoconnor** [comment]
Document this shit @CestDiego ! *cracks the whip*

File: layers/+window-management/eyebrowse/packages.el (hunk 1 above)
Comment (CestDiego) [inline layers/+window-management/eyebrowse/packages.el]: hehe

**Comment by CestDiego** [comment]
I'll document later tonight....gosh @robbyoconnor xD take it easy :P

**Comment by robbyoconnor** [comment]
*puts the whip away and gets out the bat* Bats make people work quicker for some reason...not sure why...

**Comment by robbyoconnor** [comment]
YAY! now you need to rebase @CestDiego

**Pull request closed without merging**